src/controller/articlesController.js

```diff
@@ -305,6 +297,18 @@ class ArticleController {
           article.viewStats,
           article.title
         );
+        const getTags = await ArticleTag.findAll({
+          where: {
+            articleId: article.id
+          },
+          include: [{
+            model: Tags,
+          }]
+        });
+        const getTagNames = getTags
```

Comment:
This should be `tagNames` and not `getTagNames`

Reply:
Since `getTagNames` was called once after declaration, I didn't use it again, I just matched it values directly as:
`article.dataValues.tags = tags.map(item => item
          .dataValues.Tag.dataValues.tagName);`